### PR TITLE
Switch module for Juniper EX2200 and EX4200

### PR DESCRIPTION
**Pull request by julsemaan**
- Corrects the setAdminStatus method of the Juniper module
- Adds VoIP support for 2200 and 4200
- Adds RADIUS deauthentication for 2200 and 4200
- Adds Juniper VoIP configuration to PacketFence_Network_Devices_Configuration_Guide.asciidoc

### CODE REVIEW

File: lib/pf/Switch/Juniper/EX2200.pm [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+    my $result = $this->{_sessionRead}
+        ->get_table( -baseoid => $OID_ifName );
+
+    
+    foreach my $key ( keys %{$result} ) {
+        my $portName = $result->{$key}; 
+        if ($portName eq $nas_port_id ){
+            $key =~ /^$OID_ifName\.(\d+)$/;
+            my $ifindex = $1;
+            $logger->info("Found ifindex $ifindex for nas port id $nas_port_id");
```
Comment (fdurand) [inline lib/pf/Switch/Juniper/EX2200.pm]: $logger->debug

File: lib/pf/Switch/Juniper/EX2200.pm [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+
+sub wiredeauthTechniques { 
+   my ($this, $method, $connection_type) = @_;
+   my $logger = Log::Log4perl::get_logger( ref($this) );
+
+
+    if ($connection_type == $WIRED_MAC_AUTH) {
+        my $default = $SNMP::RADIUS;
+        my %tech = (
+            $SNMP::RADIUS => \&deauthenticateMacRadius,
```
Comment (fdurand) [inline lib/pf/Switch/Juniper/EX2200.pm]: $SNMP::RADIUS => 'deauthenticateMacRadius',

**Comment by dwlfrth** [comment]
Add NEWS file entry ;)

File: lib/pf/Switch/Juniper/EX2200.pm [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+
+}
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2013 Inverse inc.
```
Comment (cgx) [inline lib/pf/Switch/Juniper/EX2200.pm]: BREAKING NEWS: we are in 2014 ;-)

**Comment by cgx** [comment]
.. and update `PacketFence_Network_Devices_Configuration_Guide.asciidoc`. Thanks!

**Comment by extrafu** [comment]
Good job!

**Pull request merged**